ui/ducks/bridge/selectors.ts

```diff
@@ -124,10 +129,40 @@ export const getToToken = (
 
 export const getFromAmount = (state: BridgeAppState): string | null =>
   state.bridge.fromTokenInputValue;
-export const getToAmount = (_state: BridgeAppState) => {
-  return '0';
+
+export const getBridgeQuotes = (state: BridgeAppState) => {
+  return {
+    quotes: state.metamask.bridgeState.quotes,
+    quotesLastFetchedMs: state.metamask.bridgeState.quotesLastFetched,
+    isLoading:
+      state.metamask.bridgeState.quotesLoadingStatus === RequestStatus.LOADING,
+  };
 };
 
+export const getRecommendedQuote = createSelector(
+  getBridgeQuotes,
+  ({ quotes }) => {
+    // TODO implement sorting
+    return quotes[0];
+  },
+);
+
+export const getQuoteRequest = (state: BridgeAppState) => {
+  const { quoteRequest } = state.metamask.bridgeState;
+  return quoteRequest;
+};
+
+export const getToAmount = createSelector(getRecommendedQuote, (quote) =>
+  quote
+    ? calcTokenAmount(
+        quote.quote.destTokenAmount,
+        quote.quote.destAsset.decimals,
+      )
+        .toFixed(3)
+        .toString()
+    : undefined,
+);
```

Comment:
Is this for display purposes only? That's how I'm reading it since `toFixed(3)` is called. If so, I think an additional comment and function name change to `getDisplayToAmount` would help readability.

Reply:
It is. I'll change this to only return the BigNumber token amount and will format the value in the component to keep display logic and data separate.